src/properties.rs

```diff
@@ -50,6 +50,7 @@ where
 
         match self.display_size {
             DisplaySize::Display128x32 => Command::ComPinConfig(false, false).send(&mut self.iface),
+            DisplaySize::Display132x64 => Command::ComPinConfig(true, false).send(&mut self.iface),
```

Comment:
I don't know what this is

Reply:
Not a very well named `Command` I'll admit. The first `bool` sets whether the display lines alternate or not. If set true, the driver IC remaps the rows like this:

![image](https://user-images.githubusercontent.com/800791/54076990-4326ca80-42aa-11e9-9f2c-41dad2d1ef9c.png)

You might want to try setting the first argument to `false` to fix your "scanline" problem described in #76. I don't know what the mapping is like for SSD1305s so YMMV.

Reply:
The second argument defines in which direction the scan occurs. This is always false, but if your display ends up upside down (without using `Rotation`), you might want to set this to `true`.